### DISCUSSION TITLE
android emulator에서 사진을 선택해도 아무런 반응이 없는 현상

### DISCUSSION THREAD

**Discussion by cotingmaster** [velopert/dealing-with-react-native#3] [Q&A] [answered]
안녕하세요 안드로이드 앱을 만들려고 하는데요 react-native-image-picker를 사용해서 사진을 에뮬레이터에서 찍는 것까진 되는데, 사진이 나타나지가 않아요. 그냥 아무 반응이 없어요. 이 부분은 동영상으로 캡처했습니다

https://user-images.githubusercontent.com/93507496/142407151-b0fd4c94-b6ec-4dab-934d-2307aaf19bf2.mov

제 노트북 사양은 m1, os는 moterey 12.0이며 사양을 캡처했습니다.
![스크린샷 2021-11-18 오후 8 17 29](https://user-images.githubusercontent.com/93507496/142407375-b077366a-80d3-45dc-9ad9-be92a40e6351.png)

안드로이드 스튜디오 사양은 캡처한 화면을 3번째 항목입니다 (pixel 4 api 29)
<img width="1019" alt="스크린샷 2021-11-18 오후 8 19 26" src="https://user-images.githubusercontent.com/93507496/142407345-04d735cd-afd4-43d9-90bd-a053ea3613aa.png">

책 466p 부터의 설명대로 코딩했고, p471 중간에 언급하신 "애플 실리콘이 탑재된 OS 사용하는 경우 이미지를 선택했을 때 아무런 반응이 없을 수 있습니다" 부분이 있어서 Xcode도 설치했습니다만 그 뒤에 열라고 하신 파일을 어떻게 열어야 할 지 모르겠더라고요. 혹시 Xcode에서 폴더 이동하는 방법 아시는 분.. 책에서 PublicGalleryJonedoe.xcworkspace 파일을 열라고 하셨는데...

**Comment by cotingmaster** [comment, marked as answer]
ㅋㅋㅋ
제가 올리고 제가 답변을 다네요. 저 이것때문에 일박 이일을 고생하고 이것저것 인터넷에 나온 것을 다 시도해봤는데 제가 잘못한 것은 css에서 이미지 태그에 높이와 너비를 주지 않았던 것이 원인이었습니다. 모두들 저같은 실수 안하시죠? ㅋㅋㅋㅋㅋ

**Comment by velopert** [comment]
고생하셨습니다.
과거에 바로 못 도와드려서 죄송합니다..! 🙏🏽
